packages/react-i18n/README.md

```diff
@@ -241,6 +241,16 @@ i18n.translate(key, {scope: 'MyComponent.option'});
 i18n.translate(key, {scope: ['MyComponent', 'option']});
 ```
 
+It may be necessary to check dynamic keys. You can use the `translationKeyExists` method to do so:
+
+```ts
+const keyExists = i18n.translationKeyExists(key);
```

Comment:
Just discovered the function, and docs. And it seems to be an error here. The `key` here must be `MyComponent.option.${key}`, right?

Also, any reason not to have the same syntax as `.translate`? Kinda weird if you have `.translate(xxx, {scope: yyy})` on one side, and something else on the other side.

cc @CameronGorrie @TheMallen